corehq/apps/app_manager/static/app_manager/js/forms/form_workflow.js

```diff
@@ -192,6 +194,26 @@ hqDefine('app_manager/js/forms/form_workflow', function () {
             return (!self.autoLink() || self.manualDatums());
         });
 
+        if (self.advancedMode) {
+            self.advancedModeDatumsElement = uiElementKeyValueList.new(
+                String(Math.random()).slice(2),  // random id
+                gettext("Manual Linking Datums"),
+                gettext("Set datums required to navigate to the selected form or menu"),
+                {key: gettext("Datum ID"), value: gettext("XPath Expression")}  // placeholders
+            );
+            let datumsDict = {};
```

Comment:
I think this can be a `const`. I don't see you reassigning it.

Reply:
I was thinking of it as something that would be mutated (ie, not a constant), but reading over [this page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const), I think I'm coming to agree with you.
> You should understand const declarations as "create a variable whose identity remains constant", not "whose value remains constant"

> Many style guides (including MDN's) recommend using const over let whenever a variable is not reassigned in its scope. This makes the intent clear that a variable's type (or value, in the case of a primitive) can never change. Others may prefer let for non-primitives that are mutated.

Javascript is frustrating sometimes.